### PR TITLE
Working with the API

### PR DESCRIPTION
**Pull request by codewanderlust**
## In this pull request I did the following;
- Stored data from a form into the Leaderboard API Service
- Retrieved the data from the API and using the refresh button I was able to display the scores on the UI.
- I updated the live demo link in the readme file.
- I added some styles to the UI

### CODE REVIEW

File: src/style.css [hunk 1]
```diff
@@ -69,3 +69,20 @@ h2 {
...
+  left: 80rem;
+  top: 10rem;
+  width: 40rem;
+}
+
+.form-position {
+  position: fixed;
+  right: 107rem;
+  top: 13rem;
+}
```
Comment (Gambit142) [inline src/style.css]: - [ OPTIONAL] Although the styling is not required for this milestone, the CSS declarations for this selector have made it difficult to see all the scores I posted to the endpoint when I retrieve it. I would suggest that you use a more responsive layout like `flexbox` or `grid` to align the children elements of this element. 👍

File: src/index.js [hunk 2]
```diff
@@ -1,25 +1,28 @@
...
 
-  const name = form.name.value;
+  const user = form.name.value;
   const score = form.score.value;
 
-  const leader = new Leaderboard(name, score);
-  UI.addLeader(leader);
-  LeaderboardStore.addLeader(leader);
+  if (!user || !score) return;
+  const newscore = {
```
Comment (Gambit142) [inline src/index.js:15]: - [ ] Great work ensuring that the _**username**_ and _**score**_ can't be empty and thus can't be posted to the API endpoint through the UI. 👏

File: src/index.js [hunk 3]
```diff
@@ -1,25 +1,28 @@
 // eslint-disable-next-line no-unused-vars
 import _ from 'lodash';
```
Comment (Gambit142) [inline src/index.js]: - [ ] Kindly remove this Eslint disabling comment. `Eslint` helps us to write clean and industry-standard codes. To fix this lint error, kindly remove this line `import _ from 'lodash';`. You are not using any `lodash` function for this project. 👍

**Review by Gambit142 — CHANGES_REQUESTED** [review]
Hi @RoyJumah 👋, 

Good job so far!

## Highlights
- All linters checks passed ✔️ 
- Commit messages are in imperative tense and are capitalized ✔️ 
- Use of `async` and `await` keywords with the `fetch` method ✔️ 
- Use of `arrow` functions ✔️ 

There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Kindly check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

- You can also consider the suggestion tagged **_[OPTIONAL]_** in the comments under the review. 👍 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

#### Please, remember to tag me in your question so I can receive the notification. You can tag me by typing @Gambit142

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Review by algerina — APPROVED** [review]
Hi @RoyJumah 
Good job so far! 

May I highlight the following:

- The repo looks professional.:heavy_check_mark:
- There are no linter errors:heavy_check_mark:
- Your project looks awesome:heavy_check_mark:

## STATUS: APPROVED :muscle::clap:

Congratulations! :tada:
Your project is complete!  It's time to merge it :shipit:



 
 ## Optional suggestions
 
 - Please check the comment under the review.

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._
Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread and tag me @algerina if something is not 100% clear.

#### _**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_
------------------------------------------------------------------------------------------------------------------------------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)
___
_If anything seems to be unclear to you, reach out and ask anything by commenting on this Pull Request and mention me as [Algerina](https://github.com/algerina), I will receive a notification and will respond._

[GitHub](https://github.com/algerina) | [LinkedIn](https://linkedin.com/in/amel-khiri) | [Twitter](https://twitter.com/AmelQahwadji)

_Reviewed by [Amel Khiri ](https://github.com/algerina)_

**Pull request merged**